backend/tests/integrated/infrastructures/load/adapters/test_repositories.py

```diff
@@ -0,0 +1,137 @@
+import pytest
+from django.db import DatabaseError
+from pytest_mock import MockFixture
+
+from infrastructures.apps.common import exceptions as common_exceptions
+from infrastructures.apps.load import models
+from modules.common import pagination as pagination_dtos
+from modules.load.domain import ports, value_objects
+from modules.load.services import queries
+from modules.load.services.queries import ports as query_ports
+from modules.load.services.queries import queries as query_dtos
+from modules.load.services.queries.ports import repositories as query_repositories
+from tests.model_factories import DataFactory
+
+from .....fakers import fake_transformed_data
+
+
+def test_django_data_domain_repository_create_method_creates_record(
+    test_django_data_domain_repository: ports.AbstractDataDomainRepository,
+):
+    # Given
+    data = [fake_transformed_data()]
+
+    # When
+    test_django_data_domain_repository.create(data)
+
+    # Then
+    assert all(
+        models.Data.objects.filter(
+            data__contains={"full_name": data.full_name}
+        ).exists()
+        for data in data
+    )
+
+
+def test_django_data_domain_repository_create_method_raises_custom_exception_on_django_exception(
```

Comment:
I'm not sure about this `_on_django_exception` naming convention. Maybe it will be better to do some generic like `on_database_exception`?
What do you think?

Reply:
Yes yes, I agree. I think I copied this _django_exception or it was your recommendation before, but I agree on database_exception.